runners/spark/src/main/java/org/apache/beam/runners/spark/util/BroadcastHelper.java

```diff
@@ -77,7 +77,7 @@ public synchronized T getValue() {
     }
 
     @Override
-    public void broadcast(JavaSparkContext jsc) {
+    public synchronized void broadcast(JavaSparkContext jsc) {
```

Comment:
After further thought, there's no real reason to synchronize this.
To avoid any weird NPE that _might_ happen (not sure how..) we can add:
`checkNotNull(value, "Broadcast value must not be null.");` before broadcasting.
As for findbugs, we can filter this class.
Sorry for that, I will gladly do it myself if you want, I should have been more decisive in the review earlier.